azext_iot/tests/test_iot_ext_int.py

```diff
@@ -1399,3 +1399,116 @@ def test_storage(self):
                      self.check('failureReason', None),
                      self.check('type', 'export'),
                      self.exists('jobId')])
+
```

Comment:
do we not have unit test for these commands?

Reply:
+1, unit tests please.

Reply:
added